policyengine_us/parameters/gov/states/mo/tax/income/deductions/mo_ss_or_ssd_deduction_allowance.yaml

```diff
@@ -15,9 +15,13 @@ metadata:
   name: mo_ss_or_ssd_deduction_allowance
```

Comment:
```suggestion
```

Reply:
could you delete all name metadata as well

Reply:
let's save this for #4210 (once we figure out the testing issue)